examples/notebooks/keras-training/common.py

```diff
@@ -0,0 +1,179 @@
+"""Provide classes to perform private training and private prediction with
+logistic regression"""
+import tensorflow as tf
+# pylint:  disable=redefined-outer-name
+import tf_encrypted as tfe
+
+
+class LogisticRegression:
```

Comment:
maybe it's time we move this guy into its own directory, say `tfe.keras.applications` ([to match](https://www.tensorflow.org/api_docs/python/tf/keras/applications))

Reply:
Not sure. I think this repo is focusing on pre-trained models: https://keras.io/applications/

Reply:
makes sense 👍